### PR TITLE
Update installation docs

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

**Comment by blowekamp** [comment]
@philipmac

File: Documentation/docs/source/installation.rst [hunk 1]
```diff
@@ -87,53 +87,48 @@ under the dev label:
...
-the `CDash dashboard <https://open.cdash.org/index.php?project=SimpleITK>`__
-in the "Nightly Packages" section.
-
-Each row on the dashboard is a SimpleITK build on a particular system,
-and if the build was successful there will be a **package icon**:
-https://open.cdash.org/img/package.png which links to the packages build
-by the system. A user may directly download the built package from such
-a link.
+If the latest SimpleITK bug fixes or features are required then
+pre-release binaries are needed. The pre-release binaries are build
```
Comment (dave3d) [inline Documentation/docs/source/installation.rst]: 'built'

**Review by dave3d — APPROVED** [review]
Approved, with word fix.

File: Documentation/docs/source/installation.rst (hunk 1 above)
Comment (blowekamp) [reply Documentation/docs/source/installation.rst]: done

**Pull request merged**